src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -165,17 +165,6 @@ public Page<PrisonerBookingSummary> getPrisonerBookingsV2(
         return bookingService.getPrisonerBookingSummary(prisonId, bookingIds, offenderNos, legalInfo, imageId, pageable);
     }
 
-    @ApiResponses({
```

Comment:
This endpoint used to be used by prison-to-probation-api-client but the code has been removed and so has not been used since 08/08/23